src/content/protocols/ethena/usde.md

```diff
@@ -0,0 +1,329 @@
+
+Users can be removed from the whitelist for minting and redeeming. If users ,that already minted `USDe`, are removed from the whitelist, they have to sell their `USDe` on the open market since they have no access to their deposited collateral which could result in the _loss of funds_ if the `USDe` price drops below the collateral value.
+
+The `USDe` staking contract which allows users to stake `USDe` includes a blacklist feature. If a _Staker_ is blacklisted, they are prevented from staking or unstaking `USDe`. This could result in _loss of funds_ if users are unable to retrieve their staked `USDe` and and if they have to sell `sUSDe` on the open market at a discount. Blacklisting can lead to a complete _loss of funds_ for a blacklisted user if their `sUSDe` is burned from their wallet and redistributed to another address or equally to all _Stakers_. Blacklisting and seizing staker funds are two steps that are carried out by two permission owners.
+
+> Upgradeability score: High
+
+## Autonomy
+
+Ethena Stablecoin Protocol diversifies into many strategies to stabilise the `USDe` price and enabling yield for stakers of `USDe`. They diversify each strategy with; 1) different custody providers, 2) different CEXs for opening hedging positions, 3) different assets holding spot, 4) different ETH staking yield source through multiple LSTs.
+
+We acknowledge that the Ethena Stablecoin Protocol diversifies interstrategically (different strategies) and intrastrategically (different assets, CEXs, custody providers, yield sources). However each strategy is carried out in a centralized manner, assets are held in custody and trading positions are openedon centralized exchanges. Diversification of exclusively centralized strategies has still high centrlization risk.
+
+> Autonomy score: High
+
+## Exit Window
+
+The actions of changing the minter role, blacklisting `USDe` stakers and seizing their funds are not protected by an _Exit Window_ and can be executed immediately, thus could harm the user immediately.
+
+> Exit Window score: High
+
+## Accessibility
+
+The user interactions with the protocol can be divided into two parts: the minting and redeeming of `USDe` and the staking of `USDe`.
+
+Users can host a frontend to mint and redeem `USDe` from https://github.com/ethena-labs/ethena-minting-client. For this action the _Accessibility_ risk is _Medium_.
+
+For staking `USDe` the protocol does not provide a frontend to self host. The staking is only possible through the Ethena frontend. For this action the _Accessibility_ risk is _High_.
+
+> Accessibility score: High
+
+## Conclusion
+
+The USDe stablecoin protocol achieves High centralization risk scores for its _Upgradeability_, _Autonomy_, _Exit Window_ and _Accessibility_ dimensions. This would mean that the protocol ranks Stage 0, however the funds are stored in centralzed custody and thus is classified as _Others_.
```

Comment:
```suggestion
The USDe stablecoin protocol achieves _High_ centralization risk scores for its _Upgradeability_, _Autonomy_, _Exit Window_ and _Accessibility_ dimensions. This would mean that the protocol ranks Stage 0, however the funds are stored in centralzed custody and thus is classified as **Others**.
```